src/main/scala/io/iohk/ethereum/healthcheck/Healthcheck.scala

```diff
@@ -1,7 +1,8 @@
 package io.iohk.ethereum.healthcheck
 
-import scala.concurrent.{ExecutionContext, Future}
-import scala.util.{Failure, Success}
+import monix.eval.Task
+
+import scala.concurrent.ExecutionContext
```

Comment:
Shouldn't all `ExecutionContext` disappear now from RPC code?

Reply:
Now it is not needed. Thanks.